### DISCUSSION TITLE
The stress_parameter_name

### DISCUSSION THREAD

**Discussion by laigner** [libAtoms/QUIP#458] [Q&A] [answered]
Hey, 
I was wondering about the `stress_parameter_name`.
I am trying to train a model on energy, forces and stress and my ASE Atoms objects contain energy forces and stress.
My question now is, because due to the discussions and issues I read it is not clear to me, is it possible to train the model directly on the stress using the `stress_parameter_name=stress` option or do I have to convert them to virials and use them as virials?

Also when I tried to use `stress_parameter_name=stress`, this keyword doesn't appear in the GAP dump training section it just told me that it found virials.

Thanks and best regards,

Lukas

**Comment by bernstei** [comment, marked as answer]
`stress_parameter_name` is used to internally create the virial, which the low-level fitting code actually uses, from the stress field.

**Reply by laigner** [reply]
Thanks for the answer. To make sure that I get it right: When I use the `stress_parameter_name=stress` the values found in the Atoms object will be converted to virials with `virials=-stress*volume` for the training process?

**Reply by jameskermode** [reply]
Yes
